conda/recipes/libcudf/meta.yaml

```
@@ -48,6 +50,9 @@ outputs:
       string: cuda{{ cuda_major }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       run_exports:
         - {{ pin_subpackage("libcudf", max_pin="x.x") }}
+      ignore_run_exports_from:
+        - nvcc_linux-64 # [linux64]
+        - nvcc_linux-aarch64 # [aarch64]
```

Can use `compiler` syntax here

```suggestion
      ignore_run_exports_from:
        - {{ compiler('cuda') }}
```